### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by stel**
I think it's a good idea to make a release that contains a new [JSONImporter](https://github.com/realm/realm-cocoa-converter/pull/16) and fixes [CSV export](https://github.com/realm/realm-cocoa-converter/pull/12).

@TimOliver, @jpsim what do you think?

### CODE REVIEW

**Comment by TimOliver** [comment]
Looks good to me! Thanks for doing that @stel! :)

File: Podfile [hunk 1]
```diff
@@ -3,14 +3,9 @@ use_frameworks!
...
     pod 'CSwiftV'
     pod 'TGSpreadsheetWriter'
-end
 
-target 'RealmConverterTests' do
-    pod 'Realm'
-    pod 'PathKit'
-    pod 'CSwiftV'
-    pod 'TGSpreadsheetWriter'
+    target 'RealmConverterTests'
```
Comment (jpsim) [inline Podfile]: 👍

File: Realm Converter iOS/Info.plist [hunk 2]
```diff
@@ -1,26 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jpsim) [inline Realm Converter iOS/Info.plist]: Was this file just not used?

Comment (stel) [inline Realm Converter iOS/Info.plist]: Yep, there is no support for iOS at all.

Comment (jpsim) [inline Realm Converter iOS/Info.plist]: 😞

**Pull request merged**